### PR TITLE
Fix fenix-qr-codes script to check differences between two sets

### PR DESCRIPTION
**Pull request by willdurand**
A quick patch to adjust some late changes in #10904 (which are/were valid).

### CODE REVIEW

File: bin/fenix-qr-codes.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 const path = require('path');
 
 const QRCode = require('qrcode');
+const config = require('config');
 
-const addonIdsWithQRCodes = require('../config/lib/addonIdsWithQRCodes');
-
-const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+const fetch = (...args) =>
+  import('node-fetch').then(({ default: fetch }) => fetch(...args));
```
Comment (willdurand) [inline bin/fenix-qr-codes.js:13]: I applied prettier 🙈

File: bin/fenix-qr-codes.js [hunk 2]
```diff
@@ -20,34 +22,36 @@ if (!fs.existsSync(destDir)) {
...
 
-  for (const addonId of addonIdsFromAPI) {
-    if (!knownIds.includes(addonId)) {
-      console.log(`'addonIdsWithQRCodes' might be outdated:`);
-      console.log(`  addonIdsWithQRCodes=${knownIds.sort()}`);
-      console.log(`  IDs from API=${addonIdsFromAPI.sort()}`);
-      process.exit(1);
-    }
+  const knownIds = config.get('addonIdsWithQRCodes');
+  const addonIdsFromAPI = [...new Set(addons.map(({ id }) => id))];
```
Comment (bobsilverberg) [inline bin/fenix-qr-codes.js:45]: Do we really need all this `Set` stuff? Can we not just use `deep-eql`?

**Review by bobsilverberg — APPROVED** [review]
Looks good. It still seems overly complicated to me, if all we need to do is check that two arrays contain the same elements, but I'll leave that up to you.

r+wc

File: bin/fenix-qr-codes.js (hunk 2 above)
Comment (willdurand) [reply bin/fenix-qr-codes.js:45]: The `Set` combined with the array spread is to end up with an array without duplicates. That's needed, whether we use `deep-eql` or not. We could use `deep-eql` a few lines below, though.

**Pull request merged**